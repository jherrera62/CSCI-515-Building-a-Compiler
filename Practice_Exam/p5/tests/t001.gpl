//repeat-until on a multiple statements with curly-braces
int counter=5;
on akey
{

  repeat {
    print("Counting down: " + counter);
    counter--;
  } until(counter==0);

}
