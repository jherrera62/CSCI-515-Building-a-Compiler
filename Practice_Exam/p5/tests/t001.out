gpl.cpp::main()
  input file(tests/t001.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int counter = 5
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[7]: Counting down: 5
print[7]: Counting down: 4
print[7]: Counting down: 3
print[7]: Counting down: 2
print[7]: Counting down: 1

User quit program. Printing the symbol table.
int counter = 0
